bin/update-wordnet-verbs

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const fs = require('fs');
+const wordnet = require('wordnet');
+
+/**
+ * Generates a compact JSON file with all english verbs from wordnet.
+ */
+
+const updateVerbs = async () => {
+  console.log(`Finding verbs..`);
+
+  await wordnet.init();
+
+  const verbs = {};
```

Comment:
Can you add a comment here explaining that we're storing the verbs as an object `{ "name": true}` so we can use it as a map (more performant comparing to an array)?

Reply:
Added a comment on the doc block of the function.